src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -21676,7 +21676,9 @@ End Namespace
             Dim libWithVTRef = libWithVT.EmitToImageReference()
 
             Dim comp = VisualBasicCompilation.Create("test", references:={libWithVTRef, corlibWithVTRef})
-            Assert.True(comp.GetWellKnownType(WellKnownType.System_ValueTuple_T2).IsErrorType())
+            Dim found = comp.GetWellKnownType(WellKnownType.System_ValueTuple_T2)
+            Assert.False(found.IsErrorType())
```

Comment:
>Assert.False(found.IsErrorType()) [](start = 12, length = 33)

Do we have a test covering an ambiguity that doesn't involve core library?

Reply:
"attribute in two libraries" scenario in GetWellKnownType test

---
In reply to: [585751950](https://github.com/dotnet/roslyn/pull/51571#discussion_r585751950) [](ancestors = 585751950)